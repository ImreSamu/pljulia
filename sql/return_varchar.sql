CREATE FUNCTION julia_varchar()
RETURNS VARCHAR AS $$
    "varchar"
$$ LANGUAGE pljulia;
SELECT julia_varchar();
DROP FUNCTION julia_varchar();
