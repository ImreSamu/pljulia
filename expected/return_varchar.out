CREATE FUNCTION julia_varchar()
RETURNS VARCHAR AS $$
    "varchar"
$$ LANGUAGE pljulia;
SELECT julia_varchar();
 julia_varchar 
---------------
 varchar
(1 row)

DROP FUNCTION julia_varchar();
